entregas/mayenSergio/carreraCamellos.java

```diff
@@ -0,0 +1,56 @@
+package entregas.mayenSergio;
+import java.util.Scanner;
+
+public class carreraCamellos {
+    public static void main(String[] args){
+
+        Scanner siguiente = new Scanner(System.in);
+        final int posicionTotal = 1;
```

Comment:
```
final int POSICION_TOTAL = 1;
```
¿Para qué utilizas esta constante? No me queda nada claro mirando el código, porque si es constante, al for de la línea 26 no le encuentro sentido.